Marlin/src/inc/SanityCheck.h

```diff
@@ -2364,13 +2364,25 @@ static_assert(   _ARR_TEST(3,0) && _ARR_TEST(3,1) && _ARR_TEST(3,2)
     #if DISABLED(Z_TRIPLE_STEPPER_DRIVERS)
       #error "Z_STEPPER_ALIGN_KNOWN_STEPPER_POSITIONS requires Z_TRIPLE_STEPPER_DRIVERS."
     #endif
-    constexpr float sanity_arr_screw_xy[][2] = Z_STEPPER_ALIGN_STEPPER_XY;
+    constexpr float sanity_arr_screw_stepper_xy[][2] = Z_STEPPER_ALIGN_STEPPER_XY;
     static_assert(
-      COUNT(sanity_arr_screw_xy) == Z_STEPPER_COUNT,
+      COUNT(sanity_arr_screw_stepper_xy) == Z_STEPPER_COUNT,
       "Z_STEPPER_ALIGN_STEPPER_XY requires three {X,Y} entries (one per Z stepper)."
     );
   #endif
 
+  // Make sure probe points are probeable
+  constexpr int sanity_arr_screw_xy[][2] = Z_STEPPER_ALIGN_XY;
+  constexpr float sanity_arr_probe_offset[3] = NOZZLE_TO_PROBE_OFFSET;
+  #define SAFEX(X) (WITHIN(X, X_MIN_POS + sanity_arr_probe_offset[0], X_MAX_POS - sanity_arr_probe_offset[0]))
+  #define SAFEY(Y) (WITHIN(Y, Y_MIN_POS + sanity_arr_probe_offset[1], Y_MAX_POS - sanity_arr_probe_offset[1]))
```

Comment:
I think you need to add the offset for both the MIN and MAX cases (rather than subtract for MAX).

Right now this would consider a point off the right side of the bed probable by a left-mounted probe.

Reply:
Maybe it makes sense to `ABS()` the offset? If we have a 10mm offset to either side we have to remain within 10mm of the bed edge, right?

Reply:
The left limit for the probe is the **larger** of `X_MIN_BED` and `X_MIN_POS + HOTEND_OFFSET_X`.
The right limit for the probe is the **smaller** of `X_MAX_BED` and `X_MAX_POS + HOTEND_OFFSET_X`.

Follow?